requirements.txt

```diff
@@ -10,6 +10,7 @@ Flask-RESTful==0.3.6
 itsdangerous==0.24
 Jinja2>=2.10.1
 MarkupSafe==1.0
+marshmallow==3.0.0rc7
```

Comment:
does marshmallow have official version other than rc?

Reply:
Previous to rc7, marshmallow released rc6, rc5, rc4, rc3, etc. It does not look like there is an alternate official version

https://marshmallow.readthedocs.io/en/3.0/changelog.html